### PR TITLE
appraisal.lic: add bundle support.

### PR DESCRIPTION
**Pull request by Sarvatt**

### CODE REVIEW

File: appraisal.lic [hunk 1]
```diff
@@ -39,6 +41,15 @@ class Appraisal
     @equipment_manager.items.each { |item| break unless appraise_item?(item) }
   end
 
+  def train_appraisal_with_bundle
+    case bput("appraise my bundle quick", 'Roundtime', 'Appraise what', 'You cannot appraise')
```
Comment (jandersson) [inline appraisal.lic]: Whats the difference between using quick and not using it?

File: appraisal.lic [hunk 2]
```diff
@@ -39,6 +41,15 @@ class Appraisal
...
+    when 'Appraise what', 'You cannot appraise'
```
Comment (jandersson) [inline appraisal.lic]: This doesn't seem necessary

File: appraisal.lic [hunk 3]
```diff
@@ -39,6 +41,15 @@ class Appraisal
...
+      return
+    else
+      waitrt?
```
Comment (jandersson) [inline appraisal.lic]:
```
bput('appraise bundle...
waitrt?
```
would accomplish the same thing, theres no need for a case/switch here

File: appraisal.lic (hunk 1 above)
Comment (Sarvatt) [reply appraisal.lic]: it's harder so should train higher

File: appraisal.lic (hunk 2 above)
Comment (Sarvatt) [reply appraisal.lic]: you're right, fixed this up.

File: appraisal.lic (hunk 3 above)
Comment (Sarvatt) [reply appraisal.lic]: this too, thanks.

**Pull request merged**